pkg/template/evaluation_ctx.go

```diff
@@ -4,9 +4,9 @@
 package template
 
 import (
+	"carvel.dev/ytt/pkg/template/core"
 	"fmt"
```

Comment:
Just a nit: "fmt" should be above "carvel.dev/ytt/pkg/template/core" and separated by a blank line.
```suggestion
	"fmt"
	
	"carvel.dev/ytt/pkg/template/core"
```